src/components/Panel.tsx

```diff
@@ -53,19 +53,30 @@ export const Panel = ({
           </Styled.Header>
         )
       )}
-      <Styled.Content className={className}>{children}</Styled.Content>
+      <Styled.Content>{children}</Styled.Content>
     </Styled.Left>
     {slotRight}
   </Styled.Panel>
 );
 
 const Styled: Record<string, AnyStyledComponent> = {};
 
-Styled.Panel = styled.section`
+Styled.Panel = styled.section<{ onClick?: () => void }>`
+  --panel-paddingY: 1rem;
+  --panel-paddingX: 1rem;
+  --panel-content-paddingY: var(--panel-paddingY);
+  --panel-content-paddingX: var(--panel-paddingX);
+
   ${layoutMixins.row}
 
   background-color: var(--color-layer-3);
   border-radius: 0.875rem;
+
+  ${({ onClick }) =>
+    onClick &&
+    css`
+      cursor: pointer;
+    `}
```

Comment:
Should we also add `hover/focus` styling here? If there is an easy way to display the arrow's `:hover` effect when the parent `<Panel>` is hovered that would be ideal.

<img width="375" alt="Screen Shot 2023-10-31 at 10 35 32 AM" src="https://github.com/dydxprotocol/v4-web/assets/13111994/cdef5841-c7e2-45b2-bb47-54f3dac62d4b">